bot/bot.go

```diff
@@ -83,6 +83,7 @@ func userWasMentioned(user *discordgo.User, mentioned []*discordgo.User) bool {
 }
 
 func (b *AIBot) messageCreate(s *discordgo.Session, m *discordgo.MessageCreate) {
+	logger := slog.Default().WithGroup("messageCreate")
```

Comment:
This and the `slog.Error("something bad", slog.Any("error", err))` invocations feel a little awkward to me, I'll revisit the slog documentation + best practices soon and see if there are any better options